packages/core/src/lib/sign-in.ts

```diff
@@ -24,8 +24,14 @@ const assignSignInResult = async (ctx: Context, provider: Provider, userId: stri
 };
 
 export const sendSignInWithEmailPasscode = async (ctx: Context, jti: string, email: string) => {
-  assertThat(!email || emailReg.test(email), new RequestError('user.invalid_email'));
-
+  assertThat(emailReg.test(email), new RequestError('user.invalid_email'));
+  assertThat(
+    await findUserByEmail(email),
```

Comment:
Looks like findUserByEmail will throw a Not Found Exception?

Reply:
Thanks, has change to `hasUserWithEmail`